redisson/src/main/java/org/redisson/connection/pool/ConnectionPool.java

```diff
@@ -131,21 +131,18 @@ public void run() {
                             if (initPromise.isDone()) {
                                 return;
                             }
-                            
```

Comment:
please remove this file from changeset

Reply:
i roll back this file to you last commit